### PR TITLE
[CHANGELOG] Add changes in buildType and SpdxOrganization

### PR DESCRIPTION
**Pull request by bact**
Update the change log with changes from
- #875
- #880
- #815
- #741

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,9 +23,16 @@
 - **Added:** `adler32` entry to `Core/HashAlgorithm` - [#826](https://github.com/spdx/spdx-3-model/pull/826)
   - The Adler-32 checksum, previously available in SPDX 2.3, has been
     reintroduced.
+- **Added:** `Core/SpdxOrganization` - [#880](https://github.com/spdx/spdx-3-model/pull/880)
+  - An `SpdxOrganization` individual, an Organization representing the SPDX
+    Project, is added. It is by definition the creator of all individuals
```
[the remark this answers is not in the document]
Comment (bact) [reply CHANGELOG.md:28]: Thank you.

I copied this from the model Markdown file - https://github.com/spdx/spdx-3-model/blob/main/model/Core/Individuals/SpdxOrganization.md - may need to update there as well?

**Comment by bact** [comment]
> Just to clarify that the organization will be used for all "Element types" which is to mean any individual with a type that subclasses Element

Thank you Gary.

I copied that text from the model Markdown file - https://github.com/spdx/spdx-3-model/blob/main/model/Core/Individuals/SpdxOrganization.md - may need to update there as well?

**Comment by goneall** [comment]
> > Just to clarify that the organization will be used for all "Element types" which is to mean any individual with a type that subclasses Element
> 
> Thank you Gary.
> 
> I copied that text from the model Markdown file - https://github.com/spdx/spdx-3-model/blob/main/model/Core/Individuals/SpdxOrganization.md - may need to update there as well?
> 
> Update: I take this opportunity to update the description there as well in one PR.

We should update the description in the SpdxOrganization.md file, but we can do that in 3.1 since we've already produced the PDF.

**Comment by zvr** [comment]
I am not sure what the difference is.

Do we ever expect to have Individuals that are not Elements? I don't think so.

**Comment by bact** [comment]
If there's no difference, does it mean we're good with what it is now?

**Comment by goneall** [comment]
> I am not sure what the difference is.
> 
> Do we ever expect to have Individuals that are not Elements? I don't think so.

Are the enum values technically individuals?

**Comment by bact** [comment]
Likely 

https://github.com/spdx/spec-parser/blob/main/spec_parser%2Frdf.py#L191-202

**Comment by goneall** [comment]
Since enums are individuals that do not use the CreationInfo/Organization, it is my opinion that we should clarify which Individuals the SpdxOrganization belongs to.

**Comment by bact** [comment]
@goneall  "Element types" added to the description

**Review by goneall — APPROVED** [review]
LGTM

**Review by zvr — APPROVED** [review]

**Pull request merged**